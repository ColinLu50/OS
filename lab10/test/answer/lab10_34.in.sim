4820
208
208
111
208
111
