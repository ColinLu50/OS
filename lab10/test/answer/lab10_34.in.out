FCFS
92 -> 30 -> 17 -> 0 -> 73 -> 56 -> 48 -> 88 -> 105 -> 36 -> 42 -> 26 -> 16 -> 71 -> 116 -> 98 -> 97 -> 56 -> 28 -> 84 -> 106 -> 24 -> 54 -> 34 -> 76 -> 53 -> 92 -> 16 -> 64 -> 34 -> 13 -> 107 -> 111 -> 59 -> 76 -> 0 -> 25 -> 53 -> 30 -> 76 -> 3 -> 109 -> 63 -> 66 -> 45 -> 36 -> 56 -> 13 -> 51 -> 25 -> 92 -> 35 -> 101 -> 73 -> 114 -> 76 -> 11 -> 113 -> 72 -> 74 -> 0 -> 10 -> 113 -> 115 -> 107 -> 71 -> 5 -> 27 -> 114 -> 58 -> 40 -> 116 -> 1 -> 45 -> 107 -> 5 -> 37 -> 78 -> 77 -> 9 -> 64 -> 11 -> 38 -> 4 -> 77 -> 57 -> 23 -> 64 -> 114 -> 101 -> 57 -> 92 -> 62 -> 101 -> 66 -> 23 -> 110 -> 48 -> 3 -> 47 -> 56 -> 103 -> 79 -> 59 -> 51 -> 86 -> 105 -> 4 -> 89 -> 11 -> 40 -> 5 -> 2 -> 68 -> 7 -> 38
4820
SSTF
92 -> 92 -> 92 -> 92 -> 89 -> 88 -> 86 -> 84 -> 79 -> 78 -> 77 -> 77 -> 76 -> 76 -> 76 -> 76 -> 74 -> 73 -> 73 -> 72 -> 71 -> 71 -> 68 -> 66 -> 66 -> 64 -> 64 -> 64 -> 63 -> 62 -> 59 -> 59 -> 58 -> 57 -> 57 -> 56 -> 56 -> 56 -> 56 -> 54 -> 53 -> 53 -> 51 -> 51 -> 48 -> 48 -> 47 -> 45 -> 45 -> 42 -> 40 -> 40 -> 38 -> 38 -> 37 -> 36 -> 36 -> 35 -> 34 -> 34 -> 30 -> 30 -> 28 -> 27 -> 26 -> 25 -> 25 -> 24 -> 23 -> 23 -> 17 -> 16 -> 16 -> 13 -> 13 -> 11 -> 11 -> 11 -> 10 -> 9 -> 7 -> 5 -> 5 -> 5 -> 4 -> 4 -> 3 -> 3 -> 2 -> 1 -> 0 -> 0 -> 0 -> 97 -> 98 -> 101 -> 101 -> 101 -> 103 -> 105 -> 105 -> 106 -> 107 -> 107 -> 107 -> 109 -> 110 -> 111 -> 113 -> 113 -> 114 -> 114 -> 114 -> 115 -> 116 -> 116
208
SCAN
92 -> 89 -> 88 -> 86 -> 84 -> 79 -> 78 -> 77 -> 76 -> 74 -> 73 -> 72 -> 71 -> 68 -> 66 -> 64 -> 63 -> 62 -> 59 -> 58 -> 57 -> 56 -> 54 -> 53 -> 51 -> 48 -> 47 -> 45 -> 42 -> 40 -> 38 -> 37 -> 36 -> 35 -> 34 -> 30 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 17 -> 16 -> 13 -> 11 -> 10 -> 9 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 0 -> 0 -> 97 -> 98 -> 101 -> 103 -> 105 -> 106 -> 107 -> 109 -> 110 -> 111 -> 113 -> 114 -> 115 -> 116
208
C-SCAN
92 -> 89 -> 88 -> 86 -> 84 -> 79 -> 78 -> 77 -> 76 -> 74 -> 73 -> 72 -> 71 -> 68 -> 66 -> 64 -> 63 -> 62 -> 59 -> 58 -> 57 -> 56 -> 54 -> 53 -> 51 -> 48 -> 47 -> 45 -> 42 -> 40 -> 38 -> 37 -> 36 -> 35 -> 34 -> 30 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 17 -> 16 -> 13 -> 11 -> 10 -> 9 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 0 -> 0 -> 116 -> 116 -> 115 -> 114 -> 113 -> 111 -> 110 -> 109 -> 107 -> 106 -> 105 -> 103 -> 101 -> 98 -> 97
111
LOOK
92 -> 89 -> 88 -> 86 -> 84 -> 79 -> 78 -> 77 -> 76 -> 74 -> 73 -> 72 -> 71 -> 68 -> 66 -> 64 -> 63 -> 62 -> 59 -> 58 -> 57 -> 56 -> 54 -> 53 -> 51 -> 48 -> 47 -> 45 -> 42 -> 40 -> 38 -> 37 -> 36 -> 35 -> 34 -> 30 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 17 -> 16 -> 13 -> 11 -> 10 -> 9 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 0 -> 97 -> 98 -> 101 -> 103 -> 105 -> 106 -> 107 -> 109 -> 110 -> 111 -> 113 -> 114 -> 115 -> 116
208
C-LOOK
92 -> 89 -> 88 -> 86 -> 84 -> 79 -> 78 -> 77 -> 76 -> 74 -> 73 -> 72 -> 71 -> 68 -> 66 -> 64 -> 63 -> 62 -> 59 -> 58 -> 57 -> 56 -> 54 -> 53 -> 51 -> 48 -> 47 -> 45 -> 42 -> 40 -> 38 -> 37 -> 36 -> 35 -> 34 -> 30 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 17 -> 16 -> 13 -> 11 -> 10 -> 9 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 0 -> 116 -> 115 -> 114 -> 113 -> 111 -> 110 -> 109 -> 107 -> 106 -> 105 -> 103 -> 101 -> 98 -> 97
111
